content/authors/jgsachen/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+# Display name
+title: Josie Sachen
+
+# Username (this should match the folder name)
+authors:
+- jgsachen
+
+weight: 22
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: Undergraduate Research Assistant
+
+# Organizations/Affiliations
+organizations:
+- name: Department of Biology
+- name: Applied Physics Laboratory
+- name: University of Washington
+
+# Short bio (displayed in user profile at end of posts)
+bio: I am interested in developing conservation solutions based on molecular ecology and genetics.
+
+interests:
+- Bioacoustics
+- Molecular Ecology 
+- Population Genetics
+- Wildlife Conversation
+
+education:
+  courses:
+  - course: BS in Molecular, Cellular, and Developmental Biology; Minor in Environmental Science and Resource Management (in progress)
+    institution: University of Washington
+    year: 2023 (expected)
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+- icon: envelope
+  icon_pack: fas
+  link: 'mailto:jgsachen@uw.edu' # For a direct email link, use "mailto:test@example.org".
+- icon: github
+  icon_pack: fab
+  link: 
+- icon: linkedin
+  icon_pack: fab
+  link: 
+- icon: paw
+  icon_pack: fas
+  link: 
```

Comment:
If you want these links to be active, fill in the link. If not, you can comment these out by #